packages/commonwealth/server/migrations/20231212140801-divastaking-community-and-contract-update.js

```diff
@@ -0,0 +1,759 @@
+'use strict';
+
+const { hasher } = require('node-object-hash');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction();
+    // Update "Communities" and "Contracts" tables as per your previous script
+    await queryInterface.sequelize.query(
+      `
+      UPDATE "Communities"
+      SET network = 'compound',
+          type = 'dao'
+      WHERE id = 'divastaking';
+    `,
+      { transaction },
+    );
+
+    await queryInterface.sequelize.query(
+      `
+      UPDATE "Contracts"
+      SET address = '0xFb6B7C11a55C57767643F1FF65c34C8693a11A70',
+          type = 'compound'
+      WHERE id = 710;
```

Comment:
Can we guarantee this id on all platforms? Or could it be diff from e.g. QA vs production?

Reply:
Resolved to identify using address instead of id